### ISSUE TITLE
install google cloud on host

### ISSUE THREAD

**Issue by Mitchellpkt** [Mitchellpkt/OperatorAware#19]
Make sure to put commands in the RunOnce script...

**Comment by Mitchellpkt** [comment]
```
File "/home/pwn_me_user/OperatorAware/operator_aware_lib/load_audio_from_filename.py", line 33, in load_audio_from_filename
    from google.cloud.speech import enums
ModuleNotFoundError: No module named 'google.cloud'
```

**Comment by Mitchellpkt** [comment]
I think will be fixed with #21
